pubber/src/main/java/udmi/lib/base/BasicPoint.java

```diff
@@ -80,11 +80,25 @@ public void setConfig(PointPointsetConfig config) {
     Value_state previousValueState = state.value_state;
     Entry previousStatus = deepCopy(state.status);
     updateStateConfig(config);
+    updateDirtyState(previousValueState, previousStatus);
+  }
+
+  private void updateDirtyState(Value_state previousValueState, Entry previousStatus) {
     dirty = dirty
         || state.value_state != previousValueState
         || !Objects.equals(state.status, previousStatus);
   }
 
+  /**
+   * Set the Intermediate State of the PointSet and update Dirty State if needed.
+   */
+  public void setIntermediateState() {
```

Comment:
revert these changes since they're no longer needed (change detection is handled at the config level, not through state?)